cli/src/cluster_query.rs

```diff
@@ -328,18 +328,23 @@ impl ClusterQuerySubCommands for App<'_, '_> {
         .subcommand(
             SubCommand::with_name("stakes")
                 .about("Show stake account information")
+                .arg(
+                    Arg::with_name("lamports")
+                        .long("lamports")
+                        .takes_value(false)
+                        .help("Display balance in lamports instead of SOL"),
+                )
                 .arg(
                     pubkey!(Arg::with_name("vote_account_pubkeys")
-                        .index(1)
                         .value_name("VOTE_ACCOUNT_PUBKEYS")
                         .multiple(true),
                         "Only show stake accounts delegated to the provided vote accounts. "),
                 )
                 .arg(
-                    Arg::with_name("lamports")
-                        .long("lamports")
-                        .takes_value(false)
-                        .help("Display balance in lamports instead of SOL"),
+                    pubkey!(Arg::with_name("withdrawer_pubkey")
+                    .value_name("WITHDRAWER_PUBKEY")
```

Comment:
nit, for consistency with others like "stake-authorize"
```suggestion
                    .value_name("PUBKEY")
```